VocaDbWeb.Core/Startup.cs

```diff
@@ -137,6 +169,7 @@ public void ConfigureContainer(ContainerBuilder builder)
 			builder.RegisterType<LaravelMixHelper>().AsSelf();
 			builder.RegisterType<Login>().AsSelf();
 			builder.RegisterType<PVHelper>().AsSelf();
+			builder.RegisterType<ViewRenderService>().As<IViewRenderService>();

```

Comment:
When should I use `SingleInstance`?

Reply:
When there should be only one instance of a class per application. If the class has no state (or you want shared state) and doesn't depend on user or culture (language), then it can usually be singleton. Singleton has better performance because of less allocations.